src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,70 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Collections;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String STRING_BUY = "buy";
+    private static final String STRING_SUPPLY = "supply";
+    private static final String STRING_СOMA = ",";
+    private static final int ZERO = 0;
+    private static final int OPERETION_INDEX = 0;
+    private static final int ONE = 1;
+    private static final int AMOUNT_INDEX = 1;
+    private static final int TWO = 2;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        File outputFile = new File(fromFileName);
+        int totalBuy = 0;
+        int totalSupply = 0;
+        totalSupply = getSupplyAndBuyFromFile(outputFile)[0];
+        totalBuy = getSupplyAndBuyFromFile(outputFile)[1];
```

Comment:
```suggestion
        List<String> data = readFromFile(inputFile)
        int[] statistic = countStatistic(data);
        int totalBuy = statistic[0];
        int totalSupply = statistic[1];
```

Reply:
store result of `countStatistic()` and reuse it, instead of calling it twice
change namings
call counting method here, instead of `readFromFile()`